lib/cli/index.js

```diff
@@ -15,37 +15,37 @@ program.option('--stack-limit <limit>', 'change the stack trace limit', parseInt
 // tasks
 
 program
-	.command('show-client-config')
+	.command('show:client-config')
```

Comment:
How about changing this to `show:config:client` ?

Reply:
@christopherobin It does require an app, see the `--app` argument 2 lines below.

Reply:
What we should try to aim at here is semantic separation by categories, I think - meaning that `-` should be a word separator and `:` should be a category separator.

With that in mind, I believe we have the following options:

  1. Leave it as is, and consider that `show should normally be a flat category (no subcategory)
  2. Classify show by sub-category (`show:server:*`, `show:client:*`). 
  3. Put all client commands under a completely separate root category `client` (`client:show:config`, `show:config`)

The question is, which one is most natural to end-users, and which one will keep making sense as we add/remove categories in the future.

Reply:
@ronkorving The thing is that AFAIK `--app` is required, if you don't pass it the command will fail. So for me `app` should instead be an argument since flags are supposed to be used for modifying execution and not providing critical data imo.

Reply:
I would add that categorization may actually suggest another option: `config:show`. Thoughts?

Reply:
How about `config:server` and `config:client` then?

Reply:
Sounds reasonable. Let's go with that.